src/server/pkg/testutil/mock_pachd.go

```diff
@@ -15,10 +16,32 @@ import (
 	"github.com/pachyderm/pachyderm/src/client/pps"
 	"github.com/pachyderm/pachyderm/src/client/transaction"
 	version "github.com/pachyderm/pachyderm/src/client/version/versionpb"
-
-	"golang.org/x/sync/errgroup"
 )
 
+// linkServers can be used to default a mock server to make calls to a real api
+// server. Due to some reflection shenanigans, mockServerPtr must explicitly be
+// a pointer to the mock server instance.
+func linkServers(mockServerPtr interface{}, realServer interface{}) {
```

Comment:
This function is used to do the forwarding from a `MockPachd` to the real underlying API servers.  Other parts of `MockPachd` do not use reflection so that developers writing tests can count on compilation errors, but this interface is internal to the testutil envs, and would not encounter compile-time type errors.